check.py

```diff
@@ -413,6 +413,7 @@ def fix(x):
     print s
     f = open('a.js', 'w')
     f.write(open(os.path.join(options.binaryen_bin, 'binaryen.js')).read())
+    f.write('var Binaryen = module.exports;\n')
```

Comment:
I see. Could we fix `check.py` to use binaryen in the normal way for a CommonJS context? It would be good if the test suite reflected real-world usage.